buildscripts/ci/withoutqt/setup.sh

```
@@ -57,18 +58,39 @@ sudo apt-get install -y --no-install-recommends "${apt_packages[@]}"
-  --slave /usr/bin/g++ g++ "/usr/bin/g++-${gcc_version}"
+  gcc_version="14"
+  $SUDO apt install -y --no-install-recommends "g++-${gcc_version}"
 
-echo export CC="/usr/bin/gcc-${gcc_version}" >> ${ENV_FILE}
-echo export CXX="/usr/bin/g++-${gcc_version}" >> ${ENV_FILE}
+  $SUDO update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-${gcc_version} 100
+  $SUDO update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-${gcc_version} 100
 
-gcc-${gcc_version} --version
-g++-${gcc_version} --version 
+  echo export CC="/usr/bin/gcc" >> ${ENV_FILE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)